cypress/utils/dashboards/data_explorer/constants.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+export const DE_INDEX_DATA = '';
```

Comment:
empty?


also instead of us needing to import all these individual sometime we can namespace stuff like

```
DE: {
  INDEX: {
     ID: 'data-explorer',
     DATA: '',
  }
...
```
so then we would have to import 3 things we just import one and use dot notation

Reply:
simplify the importing using namespace, for example:

```
export const PATHS = {
  BASE: BASE_PATH,
  ENGINE: BASE_ENGINE.url,
  SECONDARY_ENGINE: SECONDARY_ENGINE.url,
  STACK_MANAGEMENT: BASE_PATH + '/app/management',
  SECURITY_PLUGIN: BASE_PATH + '/app/security-dashboards-plugin#/',
  TENANTS_MANAGE: BASE_PATH + '/app/security-dashboards-plugin#/tenants',
  INDEX_PATTERNS: BASE_PATH + '/app/management/opensearch-dashboards/indexPatterns',
  SAVED_OBJECTS: BASE_PATH + '/app/management/opensearch-dashboards/objects',
};

```